package.json

```diff
@@ -1,8 +1,24 @@
 {
   "main": "font",
+  "author": [
+    "LukeXuan"
```

Comment:
This I assume is an oversight, and should either be removed or changed to something more generic, like `systemjs-font contributors`. If we include everyone here, which is another good alternative, I think it will become stall fairly quickly.